git-dumper.py

```diff
@@ -266,21 +276,27 @@ class FindObjectsWorker(DownloadWorker):
 
     def do_task(self, obj, url, directory, retry, timeout):
         filepath = '.git/objects/%s/%s' % (obj[:2], obj[2:])
-        response = self.session.get('%s/%s' % (url, filepath),
-                                    allow_redirects=False,
-                                    timeout=timeout)
-        printf('[-] Fetching %s/%s [%d]\n', url, filepath, response.status_code)
+        
+        if os.path.isfile(filepath):
+            printf('[-] Already downloaded %s/%s\n', url, filepath)
+            
+        if not os.path.isfile(filepath):
```

Comment:
We can use a if/else
```suggestion
        if os.path.isfile(filepath):
            printf('[-] Already downloaded %s/%s\n', url, filepath)
        else:
```